### DISCUSSION TITLE
User tokens

### DISCUSSION THREAD

**Discussion by TheBlueOompaLoompa** [DSharpPlus/DSharpPlus#866] [Support] [answered]
I know it's against ToS, but I am making a discord client with Gtk and I would like to be able to use C# because this is the easiest library for me to use, I would at least like to be able to bypass the obsolete tag if nothing else.

**Comment by Emzi0767** [comment, marked as answer]
This feature is long gone, and it's never coming back. In latest versions of the library any user token handling code has also been removed.
